config/ecs/otel-instance-metrics-config.yaml

```diff
@@ -0,0 +1,49 @@
+receivers:
+  awscontainerinsightreceiver:
+    collection_interval: 10s
+    container_orchestrator: ecs
+
+processors:
+  batch/metrics:
+    timeout: 60s
+
+exporters:
+  awsemf:
+    namespace: ContainerInsightsEC2Instance
+    log_group_name: '/aws/ecs/containerinsights/{ClusterName}/performance'
+    log_stream_name: 'instanceTelemetry/{ContainerInstanceId}'
+    resource_to_telemetry_conversion:
```

Comment:
Do we need this conversion? Do you need to add resource detector processor or it's implemented in `awscontainerinsightreceiver`

Reply:
Yea, we need this conversion to convert the array list. And I don't need any processor for now.